face_detection_masking/main.py

```diff
@@ -186,19 +185,25 @@ def process_img(
         bottom_right_y = int(bottom_right_y) + int(make_square)
 
     # Make the background color grey
-
     # naive fix for linting error: line getting to long ->
     crop_img = masked_img[top_left_y:bottom_right_y, top_left_x:bottom_right_x].copy()
     crop_img[np.where((crop_img == [0, 0, 0]).all(axis=2))] = [140, 141, 137]
 
     # Convert to Grayscale
+
     gray_img = cv2.cvtColor(crop_img, cv2.COLOR_BGR2GRAY)
 
     # resize
     final_img = resize(gray_img)
 
+    # temp directory to store images
+    if not os.path.exists(constants.TEMP_DIR[1:]):
+        os.makedirs(constants.TEMP_DIR[1:])
+
     # Save processed image to local dir
-    processed_file_path = f"{constants.TEMP_DIR}/{constants.PROCESSED_IMAGE}"
+    processed_file_path = (
+        os.getcwd() + f"{constants.TEMP_DIR}/{constants.PROCESSED_IMAGE}"
```

Comment:
likewise, if you have another constant for `tmp`, you can clean this up by moving `os.getcwd()` inside of the f-string